src/styles/variables.css

```diff
@@ -27,6 +29,7 @@
 
   /* typography */
   --font-size-m: 0.875rem;
+  --font-size-l: 1rem;
```

Comment:
oh in this PR I made `--font-size-m` be 1.4rem, which will be larger than this `--font-size-l`. Would you mind renaming it to `--font-size-s` instead to keep it consistent [with this other PR](https://github.com/contentful/experience-builder/pull/137)?